builtin/providers/aws/resource_aws_dms_replication_instance.go

```diff
@@ -174,12 +175,28 @@ func resourceAwsDmsReplicationInstanceCreate(d *schema.ResourceData, meta interf
 		return err
 	}
 
-	err = waitForInstanceCreated(conn, d.Get("replication_instance_id").(string), 30, 20)
+	//err = waitForInstanceCreated(conn, d.Get("replication_instance_id").(string), 30, 20)
+	//if err != nil {
+	//	return err
+	//}
+
+	d.SetId(d.Get("replication_instance_id").(string))
+
+	stateConf := &resource.StateChangeConf{
+		Pending:    []string{"creating"},
+		Target:     []string{"available"},
+		Refresh:    resourceAwsDmsReplicationInstanceStateRefreshFunc(d, meta),
+		Timeout:    d.Timeout(schema.TimeoutCreate),
```

Comment:
`schema.TimeoutCreate` isn't defined, are you just leveraging the system level 20 minute default?